.github/workflows/library.yml

```diff
@@ -327,7 +327,7 @@ jobs:
       - name: Install and run pyright
         if: matrix.task == 'pyright'
         run: |
-          npm install -g pyright
+          npm install -g pyright@1.1.386
```

Comment:
Any particular reason for this version?  Current version is 1.1.393.
If there's a reason for pinning, I prefer to put in a comment explaining why (I don't think an explanation is needed for keeping less than a Major version, the implication with Major version change is an API change).